### DISCUSSION TITLE
How to make uncertainty estimation?

### DISCUSSION THREAD

**Discussion by thetenthbox** [TorchEnsemble-Community/Ensemble-Pytorch#97] [Q&A] [answered]
Hi, I came across this project from this paper (https://papers.nips.cc/paper/2017/file/9ef2ed4b7fd2c810847ffa5fa85bce38-Paper.pdf). However, the docs and the code is not clear on how to make the actual uncertainty estimation for a prediction. `.predict(test)` only gives point estimation predictions. I would appreciate it if someone could help me here, as I am somewhat of a novice.

**Comment by xuyxu** [comment, marked as answer]
Hi @thetenthbox, models implemented in this package are not intended for the use of uncertainty estimation, but improving the predictive performance of a single base estimator instead.

After a rough searching over Github repos, it looks like that currently there is no package that provides a collection of methods used for uncertainty estimation. We will consider to add the support on uncertainty estimation in the feature ;-)

**Reply by thetenthbox** [reply]
Thank you very much for your quick reply. I appreciate the work you have done so far :D !
